lib/flake.nix

```diff
@@ -0,0 +1,97 @@
+{
+  description = "DevOS environment configuriguration library.";
+
+  inputs =
+    {
+      deploy.url = "github:serokell/deploy-rs";
+      deploy.inputs = {
+        utils.follows = "flake-utils";
+      };
+      devshell.url = "github:numtide/devshell";
+      flake-utils.url = "github:numtide/flake-utils/flatten-tree-system"; # TODO: pull in check api for simple flake
+      
+    }
+
+  outputs = { self, nixpkgs, deploy, devshell, flake-utils, ... }: let
+
+    inherit (nixpkgs) lib;
+
+    jobs = {
+      test = import ./tests {
+        inherit (self) lib;
+        inherit (nixpkgs) pkgs;
+      };
+      doc = import ./docs { 
+        inherit (self) lib;
+        inherit (nixpkgs) pkgs;
+      };
```

Comment:
Documenting function arguments `evalArgs`, and the generation thereof should be located here. dependeing on arbitrary `nixpkgs.pkgs` (unless proper follow flags are set on the invoking user flake).